src/eth.rs

```diff
@@ -857,6 +856,33 @@ impl<'d, T> RawHandle for EthDriver<'d, T> {
     }
 }
 
+pub struct EthFrame {
+    buf: *mut u8,
+    len: u32,
+}
+
+unsafe impl Send for EthFrame {}
+
+impl EthFrame {
```

Comment:
Not strictly necessary, but how about implementing `Deref` on `EthFrame`?

Reply:
`Deref` to `&[u8]` right? I feel like `as_slice()` is more clear than `*EthFrame`.

Reply:
You can make the same argument for everything that implements `Deref` then. Like`Bec`. Also:
* Deref should be in addition, not instead of `as_slice`
* It is `&EthFrame`, not `*EthFrame`